server/middleware/prepaids.coffee

```diff
@@ -51,6 +51,22 @@ module.exports =
       yield delighted.checkTriggerPrepaidAdded user, req.body.type
     res.status(201).send(prepaid.toObject())
 
+  put: wrap (req, res) ->
+    prepaid = yield database.getDocFromHandle(req, Prepaid)
+    if not prepaid
+      throw new errors.NotFound('User not found.')
+    startDate = req.body['startDate']
+    endDate = req.body['endDate']
+    maxRedeemers = req.body['maxRedeemers']
+    if startDate
+      prepaid.set('startDate',startDate) 
+    if endDate
+      prepaid.set('endDate',endDate)
+    if maxRedeemers
+      prepaid.set('maxRedeemers',parseInt(maxRedeemers))
```

Comment:
This seems fine, but you can also use lodash pick method like this to make the code shorter, which would only set the properties which are not undefined like this: https://github.com/codecombat/codecombat/blob/master/server/middleware/oauth.coffee#L9
For reference: https://lodash.com/docs/4.17.10#pick